docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,48 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, you can either use EESSI with Lmod or use a bash initialisation script.
+
+## Using EESSI with Lmod
+
+There are a few different scenarios where you may want to use EESSI with Lmod:
+
+1.  You already using Lmod version >= 8.6
+
+    In this case you should consider unsetting the `MODULEPATH`, as you do not really want to mix modules coming from EESSI and from your system
```

Comment:
Maybe?
```suggestion
    In this case, we recommend unsetting the `MODULEPATH`, because EESSI is not designed to mix modules coming from EESSI and from your system.
```